sdks/android/Makefile

```diff
@@ -172,11 +172,25 @@ build-%: package
 	cd $(APK_DIR) && $(NDK_BUILD)
 	cd $(APK_DIR) && $(ANT) debug
 
+ifneq ($(MONO_WAIT_LLDB),)
```

Comment:
This involves quite some duplicate code and targets, you can reduce it by doing something like https://gist.github.com/luhenry/a06662c281e0f2334d4164f0d20e9aab